packages/protobuf-21/PKGBUILD

```diff
@@ -56,11 +57,22 @@ build() {
   PROTOC="$srcdir"/build/protoc python -m build --wheel --no-isolation
 }
 
-package() {
+package_protobuf-21() {
   provides=(libprotoc.so libprotobuf.so libprotobuf-lite.so)
   DESTDIR="$pkgdir" cmake --install build
   cd $_name-$pkgver
   install -vDm 644 LICENSE -t "$pkgdir"/usr/share/licenses/$pkgname
   mv "$pkgdir"/usr/bin/protoc{,-21}
   ln -rs "$pkgdir"/usr/lib/$pkgname/*.so.* "$pkgdir"/usr/lib
 }
+
+package_python-protobuf-21() {
+  pkgdesc='Python bindings for Google Protocol Buffers (legacy)'
+  provides=(python-protobuf=$pkgver)
```

Comment:
This means if a user of our repo installs `python-protobuf` (or anything depending on that), they will be asked which package to install. Maybe we should remove the `provides` here but it doesn't feel wrong IMO :D

Reply:
Is there maybe an example in the official repos of a package providing an older version of a library for compatibility? If yes, what do they do?

Reply:
Good point, just checked some examples and `provides` is only added for sonames, where makepkg adds the correct version and multiple versions coexist. Otherwise, no `provides`, I'll remove it.
Examples:
- https://gitlab.archlinux.org/archlinux/packaging/packages/allegro4/-/blob/main/PKGBUILD
- https://gitlab.archlinux.org/archlinux/packaging/packages/webrtc-audio-processing-0.3/-/blob/main/PKGBUILD
- https://gitlab.archlinux.org/archlinux/packaging/packages/botan2/-/blob/main/PKGBUILD
- https://gitlab.archlinux.org/archlinux/packaging/packages/haskell-vty5/-/blob/main/PKGBUILD

Reply:
removed in 979c4979f450454ae5848f38162a91fc574ec109